ManageWikiSettings.php

```diff
@@ -1187,6 +1187,15 @@
 		'help' => 'This allows to set whether or not it should be possible for anonymous visitors of the wiki to dismiss the site notice shown.',
 		'requires' => [],
 	],
+	'wgDiscordAvatarUrl' => [
+		'name' => 'Discord Notification Avatar URL',
+		'from' => 'mediawiki',
+		'type' => 'text',
+		'overridedefault' => false,
```

Comment:
Should be '' to match  LS I think.  Not a huge deal just what I think it should be.

Reply:
Fixed it.

Reply:
Thanks!